metering/Containerfile

```diff
@@ -0,0 +1,38 @@
+ARG IMAGE_VERSION=latest
+FROM python:${IMAGE_VERSION}
+
+ARG USER_ID=45000
+ARG GROUP_ID=45000
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+USER root
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+        git \
+    && git clone https://github.com/cloudandheat/odoo_openstack_metering.git \
```

Comment:
git clone .. /opt/app ? This way we do not need the COPY in line 25.

Reply:
Thanks. changed that